### PR TITLE
Updating dashboards and documentation

### PR DESCRIPTION
**Pull request by duncan-ncc**
Pending Query_url fix https://github.com/elastic/kibana/issues/63924

### CODE REVIEW

File: docs/chapter4.md [hunk 1]
```diff
@@ -17,7 +17,12 @@ In a web browser, navigate to ```https:\\your_Linux_server``` and authenticate w
 
 First you will need to create an index. If you would like to use the pre-made dashboards, the import process creates the index for you.
 
-The dashboard and visualisation objects are contained within a NDJSON file (previously JSON) and can be easily imported by clicking ‘Management’ -> ‘Saved Objects’. Please follow the steps in Figure 1, and the NDJSON files are located in [Chapter 4 Files\dashboard 'version'.ndjson](/Chapter%204%20Files/).
+The dashboard and visualisation objects are contained within a NDJSON file (previously JSON) and can be easily imported 
+***IMPORTANT: You must open the dashboard file (in notepad or similar) and use find and replace to change all instances of "ChangeThisDomain" to your kibana URL:***
```
Comment (lllama) [inline docs/chapter4.md:21]: Looks like the trailing slash is the wrong direction?

File: docs/chapter4.md [hunk 2]
```diff
@@ -17,7 +17,12 @@ In a web browser, navigate to ```https:\\your_Linux_server``` and authenticate w
 
 First you will need to create an index. If you would like to use the pre-made dashboards, the import process creates the index for you.
 
-The dashboard and visualisation objects are contained within a NDJSON file (previously JSON) and can be easily imported by clicking ‘Management’ -> ‘Saved Objects’. Please follow the steps in Figure 1, and the NDJSON files are located in [Chapter 4 Files\dashboard 'version'.ndjson](/Chapter%204%20Files/).
+The dashboard and visualisation objects are contained within a NDJSON file (previously JSON) and can be easily imported 
+***IMPORTANT: You must open the dashboard file (in notepad or similar) and use find and replace to change all instances of "ChangeThisDomain" to your kibana URL:***
+* If you use a hostname to access kibana then it should be ```https://kibanahostname.example.com\``` where kibanahostname.example.com is your kibana DNS name
+* If you use an ip to access kibana ```https://127.0.0.1\``` where 127.0.0.1 is the ip of your kibana
```
Comment (lllama) [inline docs/chapter4.md:23]: Looks like the trailing slash is the wrong direction?

**Review by lllama — APPROVED** [review]
LGTM :shipit:

**Review by jordan-ncc — APPROVED** [review]
Have tested the changes on a testing machine and worked fine. Happy for merge

**Pull request merged**